### ISSUE TITLE
Drawing gcode looks okay, but cutting gcode is bad

### ISSUE THREAD

**Issue by KuCZik** [arpruss/gcodeplot#39]
Inkscape version 1.2.2
GCodePlot version 0.12
Viewing in CAMotics version 1.2.0

Same file, text is set to path in both scenarios, only change is between drawing/cutting in selection before saving. Is it bug, or am I doing something wrong? It is not problem of CAMotics, if I send it to cut on Klipper managed Ender 3 Pro, I get the same result.

Drawing GCode:
![image](https://user-images.githubusercontent.com/1360031/212576143-4de3bbae-4245-4d99-a7e9-da3121962b2b.png)

Cutting GCode:
![obrázek_2023-01-16_013535228](https://user-images.githubusercontent.com/1360031/212576102-24f581cb-9925-4786-8b20-bfaccf1a65a2.png)

**Comment by arpruss** [comment]
That's the drag knife handling. Are you using a drag knife?

**Comment by KuCZik** [comment]
Yes, this type 
![image](https://user-images.githubusercontent.com/1360031/212641989-3579bde8-0617-4e20-b859-0fc23fbbde5b.png)

Mounted like this
![image](https://user-images.githubusercontent.com/1360031/212642730-26550d70-40fa-4bc7-963c-489fd1029899.png)

I have it in rubber band mount to have spring action. But the real result looks like this
![image](https://user-images.githubusercontent.com/1360031/212650228-37ccae90-f161-42da-a350-d78396f1ebe6.png)

And my config is
![image](https://user-images.githubusercontent.com/1360031/212650364-7ff838fa-c3a2-45f6-aa6e-9fd87de3fdf6.png)

**Comment by arpruss** [comment]
The default settings are for an angled blade whose cutting edge lags behind the center of movement -- a drag knife. You have a what looks like a bit with a sharp point centered on the center of movement. There should be settings under Cutting Settings to turn off the drag support. But I really know very little about cutting.

**Comment by KuCZik** [comment]
No, I have a drag knife, inside is angled blade that is rotating, not a bit

**Comment by arpruss** [comment]
Weird. My feeling is that it is not lifting up enough during the weird squiggles. Is there a parameter to adjust that in the Cutting Settings?

**Comment by KuCZik** [comment]
In cutting settings I see just this
![image](https://user-images.githubusercontent.com/1360031/212732318-08d8e39c-bd7b-4308-b2f9-4403c790a86b.png)

**Comment by arpruss** [comment]
Does the Gcode include a lift for the loopy bits?

**Comment by KuCZik** [comment]
This is the GCode (I just edited in gcodeplot.py start and end GCode, edited G00 and G01 commands to G0 and G1 - Klipper does not know what GOO and G01 is, and removed parking - made my own):

[gcodeplot.py.txt](https://github.com/arpruss/gcodeplot/files/10428463/gcodeplot.py.txt)
[test.gcode.txt](https://github.com/arpruss/gcodeplot/files/10428465/test.gcode.txt)

**Comment by polhomarkho** [comment]
Hello, late to the party but wanted to add I had the same issue at first.  
The drag knife blade has an offset to the rotating axis (mine is 0.45mm) and the "tool offset" value in your config is probably too high at 1mm. You can lower it and with some trial and error you'll find the best value for your drag knife.
